src/current/v23.1/goldengate.md

```diff
@@ -518,3 +518,524 @@ Run the steps in this section on a machine and in a directory where Oracle Golde
 - [Schema Conversion Tool](https://www.cockroachlabs.com/docs/cockroachcloud/migrations-page)
 - [Change Data Capture Overview]({% link {{ page.version.version }}/change-data-capture-overview.md %})
 - [Third-Party Tools Supported by Cockroach Labs]({% link {{ page.version.version }}/third-party-database-tools.md %})
+---
```

Comment:
What's going on here?

Reply:
A really bad copy-paste error -- sorry about that.